Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull.py

```diff
@@ -32,6 +33,20 @@ class Pull(Command):
     aliases = ['up', 'update']
     help = 'Update the current checkout, synchronize git-svn if configured'
 
+    @classmethod
+    def parser(cls, parser, loggers=None):
+        parser.add_argument(
+            '-i', '--issue', '-b', '--bug', '-r',
+            dest='issue', type=str,
+            help='Number (or name) of the issue or bug to create branch for',
+        )
```

Comment:
Where did this come from? Seems unrelated

Reply:
Thought I deleted that already, removing it now...